test/Extension.Tests.ps1

```diff
@@ -15,6 +15,7 @@ Describe 'SecretManagement.LastPass tests' {
     It 'Can store a string secret which is treated like a securestring' {
         $secretText = 'This is my string secret'
         Set-Secret -Name $secretName -Vault $VaultName -Secret $secretText
+        Sync-LastPassVault -Vault $VaultName
```

Comment:
I also had fluke in my tests in a more consistend manner. 
Adding a sync fixed the problem 100% of the time.

We do a sync in Set-Secret too so I wonder if that's not in the way we pick up the secret through Get-SecretInfo and / or there's a processing delay and the second sync simply give some more time to do the deed.

Reply:
hmmm I remember seeing this bug where a newly created secret didn't meet the regex we were using... I thought I fixed it but maybe I missed a case.

Reply:
@TylerLeonhardt Yeah, I remember that.

Hmmm... I had that problem initially that if I didn't do an explicit sync in Set-Secret after setting the secret, I would create a duplicate if I create another secret right afterward. 

So I added a `Invoke-lpass 'sync'` at the end of `Set-Secret` and it  resolved the problem.
Now, if I call `Set-Secret` multiple time in a row, it seem to work...

Although.

Using this, I get duplicate created at some point.
This make sense, if we have that kind of behavior in the tests (sync lagging behind) then at some point, we'll get a `Set-Secret` that will not get anything when calling `lpass show` and will create a secret instead of editing the one present, which lead to duplicate.

```

$Vault = @{Vault = 'lpt'; Name = 'ABCDEF' }
1..10 | %{
    Set-Secret @Vault -Secret 'EDCBA' -Verbose 
    Get-Secret @Vault -AsPlainText
    Remove-Secret @Vault 
}


# View duplicate
Get-SecretInfo @vault 
# Cleanup
Get-SecretInfo @vault | % {Remove-Secret -Name $_.Name -Vault $Vault.Vault}

```

As for your concern regarding something missing with `Get-SecretInfo`, I do not think so. 
`Get-Secret` is really simple in comparison and does not filter out secrets based on some pattern.
It does exhibit the same behavior.

Reply:
I don't necessarily see this PR held back because of this though since it is not tied to a change within it. 
I have opened a separate issue : https://github.com/TylerLeonhardt/SecretManagement.LastPass/issues/19#issue-744433733